pkg/helper/client.go

```diff
@@ -126,6 +126,12 @@ func ApplyDesiredState(client client.Client, desiredState shared.State) (string,
 	// working fine after apply
 	probes := probe.Select(client)
 
+	desiredState, err := applyDefaultVlanFiltering(desiredState)
```

Comment:
If not doing it in the webhook as suggested [here](https://github.com/nmstate/kubernetes-nmstate/pull/793#issuecomment-896573208), I would at least update the enactment as well. So it will be more obvious that the bridge default value are desired and there is no diff between the desired and the current state.

Reply:
Done.
I have added a webhook that adds the default vlan configuration to the NNCP, so both NNCP and NNCEs have the desiredState with default values
I'm still adding the defaults in the Reconcile to make sure old NNCPs have the defaults after nmstate update.

We could also update the NNCP during the Reconcile, which I'm not doing in the current state of this PR, because I'm not convinced it's worth the API load this would trigger - IIUC, each node would reconciled each NNCP twice in cases when the NNCP would be updated right?

Reply:
On second thought, given that the webhook should add the defaults in majority of cases, I suppose we can check the default in Reconcile and update the policy case it's needed

Reply:
@AlonaKaplan 
After discussion with Petr, we decided it's not needed nor desired to include internal defaults into NNCP. The reason for this being that there are many various defaults set by the system as well.

The current change includes the defaults vlan setting into the NNCE to avoid the discrepancy between desiredState and NNS (since the 'default' we set for VLANs are not really system defaults).